### PR TITLE
fix: repair incomplete responses

### PR DESCRIPTION
**Pull request by victormendoza96**
remove fields in oneclick status transaction:
 'sessionId'
 'expirationDate' 
 
add fields 'balance, prepaid_balance' in oneclick and "transacción completa"

### CODE REVIEW

File: src/TransaccionCompleta/Responses/TransactionDetail.php [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+use Transbank\Utils\Utils;
+
 class TransactionDetail extends \Transbank\Webpay\WebpayPlus\Responses\TransactionDetail
 {
+    public $prepaidBalance;
+
+    public static function createFromArray(array $array)
+    {
+        $result = parent::createFromArray($array);
+        $result->prepaidBalance = Utils::returnValueIfExists($array, 'prepaid_balance');
```
Comment (Matiasnickolas) [inline src/TransaccionCompleta/Responses/TransactionDetail.php:14]: Here you are adding dinamically a _prepaidBalance_ property to result object. It isn't recommended, and in php 8 this wil throw a warning

File: src/TransaccionCompleta/Responses/TransactionDetail.php [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 namespace Transbank\TransaccionCompleta\Responses;
 
+use Transbank\Utils\Utils;
+
 class TransactionDetail extends \Transbank\Webpay\WebpayPlus\Responses\TransactionDetail
 {
+    public $prepaidBalance;
```
Comment (Matiasnickolas) [inline src/TransaccionCompleta/Responses/TransactionDetail.php:9]: This attribute is never used

File: src/TransaccionCompleta/Responses/TransactionStatusResponse.php [hunk 3]
```diff
@@ -10,11 +10,13 @@ class TransactionStatusResponse
     use HasTransactionStatus;
 
     public $vci;
+    public $prepaidBalance;
 
     public function __construct($json)
     {
         $this->vci = Utils::returnValueIfExists($json, 'vci');
         $this->setTransactionStatusFields($json);
+        $this->prepaidBalance = Utils::returnValueIfExists($json, 'prepaid_balance');
```
Comment (Matiasnickolas) [inline src/TransaccionCompleta/Responses/TransactionStatusResponse.php:19]: Here you are adding dinamically a prepaidBalance property to result object. It isn't recommended, and in php 8 this wil throw a warning

File: src/Webpay/WebpayPlus/Responses/TransactionDetail.php [hunk 4]
```diff
@@ -30,6 +31,7 @@ public static function createFromArray(array $array)
         $result->installmentsAmount = Utils::returnValueIfExists($array, 'installments_amount');
         $result->commerceCode = Utils::returnValueIfExists($array, 'commerce_code');
         $result->buyOrder = Utils::returnValueIfExists($array, 'buy_order');
+        $result->balance = Utils::returnValueIfExists($array, 'balance');
```
Comment (Matiasnickolas) [inline src/Webpay/WebpayPlus/Responses/TransactionDetail.php:34]: Here you are adding dinamically a _balance_ property to result object. It isn't recommended, and in php 8 this wil throw a warning.

File: src/Webpay/WebpayPlus/Responses/TransactionDetail.php [hunk 5]
```diff
@@ -17,6 +17,7 @@ class TransactionDetail
     public $installmentsAmount;
     public $commerceCode;
     public $buyOrder;
+    public $balance;
```
Comment (Matiasnickolas) [inline src/Webpay/WebpayPlus/Responses/TransactionDetail.php:20]: This attribute is never used

File: src/TransaccionCompleta/Responses/TransactionStatusResponse.php (hunk 3 above)
Comment (victormendoza96) [reply src/TransaccionCompleta/Responses/TransactionStatusResponse.php:19]: is not  dinamically is the same like vci property

**Review by mastudillot — APPROVED** [review]
LGTM

**Pull request merged**